src/utils/currency.ts

```diff
@@ -0,0 +1,35 @@
+import round from 'lodash/round'
+import BigNumber from 'bignumber.js'
+
+export const TICKER = 'HNT'
+export const SECURITY_TKN = 'ST'
+export const DATA_CREDIT = 'DC'
+
+BigNumber.config({
+  EXPONENTIAL_AT: [-10, 20],
+  FORMAT: {
+    decimalSeparator: '.',
+    groupSeparator: ',',
+    groupSize: 3,
+    secondaryGroupSize: 0,
+    fractionGroupSeparator: ' ',
+    fractionGroupSize: 0,
+  },
+})
+
+export const abbrFormat = (num: number) => {
+  let formatted
+  if (num % 1 !== 0) formatted = `${round(num, 2)}`
+  if (num > 999) formatted = `${round(num / 1000, 1)}k`
+  if (num > 999999) formatted = `${round(num / 1000000, 1)}m`
+  if (num > 999999999) formatted = `${round(num / 1000000000, 1)}b`
+  return formatted
+}
+
+export const currencyFormat = (bones: number, maxDecimalPlaces?: number) => {
```

Comment:
I think all currency formatting should be handled through the `Balance#toString` method in helium-js. If any fields are missing the conversion to `Balance` objects we should fix that